lib/tasks/occupations.rake

```diff
@@ -0,0 +1,46 @@
+namespace :occupations do
+  task import: :environment do
+    filename = "#{Rails.root}/lib/tasks/files/occupations.csv"
```

Comment:
I don't think I did a good job emphasizing what I meant by this from https://github.com/WorkHands/RAPiDSkills/issues/34:

>> We essentially want to pull everything from this page and periodically update it

I'd imagined we'd have a scheduled rake task that downloaded that file and re-ran this monthly or so. It's actually better that you just did this as it's 100% fine for the MVP, and this list doesnt change super often so we're good for a while.

Can you just set up a ticket for this so we don't forget this a few months from now? 

"The system downloads the list of apprenticeable occupations each month and updates the core list as needed" or something like that?

I don't know if we can just download a file to Heroku and run this task on the fly. Do you? We may need to download/upload to an S3 bucket before analyzing the file each time. Setting up that infrastructure's another solid reason for punting on this for now to make progress elsewhere.

Reply:
Ah yes, sorry, I did understand but thought that was for a task down the road. I think we can store data in the `/tmp` directory on Heroku (with the understanding that it is not guaranteed to be there later).  Do you want me to give it a go with pulling the data and importing it from the downloaded file instead?